server/services/v1/admin.go

```diff
@@ -74,6 +74,30 @@ func (a *adminService) CreateNamespace(ctx context.Context, req *api.CreateNames
 	}
 }
 
+func (a *adminService) ListNamespaces(ctx context.Context, _ *api.ListNamespacesRequest) (*api.ListNamespacesResponse, error) {
+	namespaces, err := a.tenantMgr.ListNamespaces(ctx, a.txMgr)
+	if err != nil {
+		return nil, err
+	}
+	if namespaces != nil {
```

Comment:
avoid else by simply short-circuiting to `if namespaces == nil {return ...}`

Reply:
trimmed.